src/WHAT_TO_DO_NEXT.md

```diff
@@ -0,0 +1,135 @@
+# What To Do Next
+
+## Initialize the Project
+
+To initialize a new PowerShell module, please run `Initialize-PSPackageProject -ModuleName MyTestMod -ModuleBase d:\temp\MyTestMod`.
+
+The command will generate a scaffolding for the PowerShell module called `MyTestMod` at `d:\temp\MyTestMod`.
+
+## Generated project scaffold
+
+The following project structure is generated:
+
+```PowerShell
+PS D:\> dir D:\temp\MyTestMod\
+
+
+    Directory: D:\temp\MyTestMod
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+d----           7/29/2019  4:08 PM                .ci
+d----           7/29/2019  4:08 PM                help
+d----           7/29/2019  4:08 PM                src
+d----           7/29/2019  4:08 PM                test
+-a---           7/29/2019  3:45 PM           3032 build.ps1
+-a---           7/29/2019  4:08 PM            157 pspackageproject.json
+```
+
+`build.ps1` can be used to build and test the module.
+
+Build the module: `build.ps1 -Build`
+Test the module: `build.ps1 -Test`
```

Comment:
It looks like `build.ps1 -UpdateHelp` will also work -- worth documenting here?

Reply:
Did not work when i tried yesterday :(